docs/docs/api/Tasks/getTasks.md

```diff
@@ -16,9 +16,14 @@ This is a **protected route**, a **valid JWT is required** in the header field
 #### Header
 
 ```
-Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
+Cookie: token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
+#### Parameters
+
+start: yyyymmdd\
```

Comment:
The backward slash isn't needed here

Reply:
Upon removal, everything moves to a single line.
I have removed the backward slash, and this is how it looks:
![image](https://user-images.githubusercontent.com/70231862/163821129-53891bf4-7f58-4670-a08e-6898bad23cd9.png)

Reply:
Changes are reflected in recent commits